src/components/courseReview/course/courseReviews/reviewItem/ReviewReplies.vue

```diff
@@ -300,6 +300,18 @@ const handleDeleteReply = async (repyId: number) => {
   }
 }
 
+const route = useRoute()
+
+
+onMounted(async () => {
+  await nextTick()
+  const hash = route.hash
+  if (hash && hash.includes('reply-')) {
```

Comment:
If you want to introduce the `reply-`, we need to find another way to parse the hash, instead of simply splitting the hash string by `-`.